app/__init__.py

```diff
@@ -1,7 +1,7 @@
 from flask import Flask
 from .db import db, migrate
 from .models import book
-from .routes.book_routes import books_bp
+from .routes.book_routes import bp as books_bp
```

Comment:
Using `as` to rename is one way to approach this. My preferred way is to justimport the model here, then use the qualified name where we need to access `bp`.

So here, we would
```py
from .routes import book_routes
```

Then below, we would
```py
    app.register_blueprint(book_routes.bp)
```

I'm fine with settling on `as` as the "official" way that moves forward through the remaining branches (`as` renaming is very useful), but what would you think about discussing both approaches in the text.

Reply:
I agree that it'd be useful to touch on both styles in the text and pick one to use through the project. Students don't have much if any exposure to renaming with `as` until this point, so I think I would like to keep that as what we show in the branches, but I don't feel very strongly about this if other instructors would also like to import the file then access `bp` from there.

Reply:
I agree that the benefit of seeing `as` is a strong point in its favor, so having that as the "official" version makes sense. I'll be happy if we can discuss the other way in the text, just because it's another useful way to manage imports, so the discussion wouldbe relevant to have in the same space.

Reply:
I added a note to the lesson 7 card for this.